presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -537,6 +540,117 @@ private void runTimestampQueries(String tableName, List<TimestampAndPrecision> d
         onPresto().executeQuery("DROP TABLE " + tableName);
     }
 
+    @Test(dataProvider = "storageFormatsWithNanosecondPrecision", groups = STORAGE_FORMATS)
```

Comment:
Pre-existing, but would be good to have `storageFormatsWithNanosecondPrecision` be defined in terms of storageFormats + exclusions, eg

```java
@DataProvider
public static Object[][] storageFormatsWithNanosecondPrecision()
{
    return Stream.of(storageFormats())
            .map(array -> (StorageFormat) getOnlyElement(Arrays.asList(array)))
            .filter(storageFormat -> !storageFormat.getName().equals("AVRO"))
            .map(storageFormat -> new Object[] {storageFormat})
            .toArray(Object[][]::new);
}
```